modules/react/menu/lib/MenuItem.tsx

```diff
@@ -40,7 +48,56 @@ export interface MenuItemProps {
   'aria-disabled'?: boolean;
 }
 
-const StyledItem = styled(Box.as('button'))<StyledType>(
+export interface StyledMenuItemProps {
```

Comment:
Should we extend `Box` props in the item?

Reply:
I like that idea. Added